rgf/sklearn.py

```diff
@@ -369,12 +369,15 @@ class RGFClassifier(BaseEstimator, ClassifierMixin):
 
     n_jobs : integer, optional (default=-1)
         The number of jobs to use for the computation.
-        If 1 is given, no parallel computing code is used at all.
-        If -1 all CPUs are used.
+        The substantial number of the jobs is limited by classes_.
+        If classes_ = 2, the substantial max number of the jobs is one.
+        If classes_ >= 3, the substantial max number of the jobs is the same as
```

Comment:
I think that these 2 lines we may merge into one.
Before:
> The substantial number of the jobs is limited by classes_. 
>If classes_ >= 3, the substantial max number of the jobs is the same as classes_. 

After:
If classes_ > 2, the substantial max number of the jobs is limited by classes_.

Reply:
English is difficult :smile: . Here, I want not to say classes_ equals n_jobs.
But I want to tell users:
> substantial max number of the jobs dependents on classes_.

So I'll change here to
```
The substantial number of the jobs dependents on classes_.
If classes_ = 2, the substantial max number of the jobs is one.
If classes_ >= 3, the substantial max number of the jobs is the same as
```

Reply:
Many conditions => difficult to describe them in short =(

Good idea! But I think that `>= 3` better change to `> 2` since above we have `= 2`. What do you think?